capstone/src/main/webapp/script.js

```diff
@@ -12,11 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+window.addEventListener('DOMContentLoaded', (event) => {
+  if (document.getElementById('map')) {
+    createMap();
+  }
+});
+
+var map;
```

Comment:
I'm wondering if there's a way we can encapsulate this, and then export the functions. If it's too much trouble, I think it's best practice to indicate that these are private variables by appending a _ to them.